spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -739,17 +739,12 @@ def fixture_record(fixture_name, indexer: @indexer)
       let(:date_9999_marc) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [date_9999_008, p260], 'leader' => leader)) }
       let(:date_199u_marc) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [date_199u_008, p260], 'leader' => leader)) }
       let(:not_ceased_marc) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [not_ceased_008, p260], 'leader' => leader)) }
-      let(:ceased_marc) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [ceased_008, p260], 'leader' => leader)) }
       let(:no_trailing_date_marc) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [ceased_008, p260_complete], 'leader' => leader)) }
 
       it 'displays 264 tag sorted by indicator2' do
         expect(@sample34['pub_created_display']).to eq ['[Paris] : Les Films de La Pleiade, 1956-1971.', '[Brooklyn, N.Y.] : Icarus Films, [2017]', 'Â©1956-1971']
       end
 
-      it 'displays when 008-6 is d and an end date is present in the 008' do
```

Comment:
I rewrote this test in Rust (in publication.rs) as part of troubleshooting, so I removed it here.